sw/device/silicon_creator/rom/data/rom_e2e_testplan.hjson

```diff
@@ -924,7 +924,7 @@
             '''
       tags: ["rom", "no_dv", "fpga", "silicon"]
       stage: V2
-      tests: []
+      tests: ["//sw/device/silicon_creator/rom/e2e:rom_e2e_asm_watchdog"]
```

Comment:
@timothytrippel should we leave this empty?

Reply:
yes, leave it empty. same reason as [here](https://github.com/lowRISC/opentitan/pull/15779#discussion_r1009929074).

Reply:
Done, thanks for catching that.